File: sample/src/main/java/com/moczul/sample/AndroidLogger.java
package com.moczul.sample;

import android.util.Log;

import androidx.annotation.NonNull;

import com.moczul.ok2curl.logger.Logger;

public class AndroidLogger implements Logger {

    public static final String TAG = "Ok2Curl";

    @Override
    public void log(@NonNull String message) {
        Log.v(TAG, message);
    }
}

File: sample/src/main/java/com/moczul/sample/AndroidLogger.kt
package com.moczul.sample

import android.util.Log
import com.moczul.ok2curl.logger.Logger

class AndroidLogger : Logger {

    override fun log(message: String) {
        Log.v(TAG, message)
    }

    companion object {
        const val TAG = "Ok2Curl"
    }
}